src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1288,11 +1287,25 @@ MavlinkReceiver::handle_message_odometry(mavlink_message_t *msg)
 		odometry.pitchspeed = odom.pitchspeed;
 		odometry.yawspeed = odom.yawspeed;
 
-		//TODO: Apply rotation matrix to transform from body-fixed NED to earth-fixed NED frame
+		/* the linear velocity's covariance needs to be transformed to the local frame FRD*/
```

Comment:
Seems like this was missing before (and therefore wrong/a bug). Can you make this it's own commit?